tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -1571,9 +1565,11 @@ class MklQuantizedConv2DOp
         output_max->flat<float>()(0) = max_output_value;
       } else {
         size_t depth = min_filter.NumElements();
-        AllocateOutputSetMklShape(context, 1, &output_min, {depth},
+        AllocateOutputSetMklShape(context, 1, &output_min,
+                                  {static_cast<ptrdiff_t>(depth)},
```

Comment:
I'm curious. Why does this have to be `ptrdiff_t`?

Reply:
error C2398: Element '1': conversion from 'std::size_t' to 'std::ptrdiff_t' requires a narrowing conversion

Reply:
Thank you!